src/test/scala/com/singlestore/spark/IntegrationSuiteBase.scala

```diff
@@ -146,10 +143,11 @@ trait IntegrationSuiteBase
   }
 
   def jdbcOptions(dbtable: String): Map[String, String] = Map(
-    "url"      -> s"jdbc:mysql://$masterHost:$masterPort",
-    "dbtable"  -> dbtable,
-    "user"     -> "root",
-    "password" -> masterPassword
+    "url"               -> s"jdbc:mysql://$masterHost:$masterPort",
+    "dbtable"           -> dbtable,
+    "user"              -> "root",
+    "password"          -> masterPassword,
+    "pushDownPredicate" -> "false"
```

Comment:
Why this option is added?

Reply:
In tests, we compare our results after pushdown with the Spark JDBC driver. In the new versions of Spark, the Spark JDBC driver generates invalid SingleStore queries. This option fixes Spark JDBC driver incompatibility by forcing it to generate simpler queries.